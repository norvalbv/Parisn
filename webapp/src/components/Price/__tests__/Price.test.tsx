import { FC } from 'react';
import renderer from 'react-test-renderer';
import PriceDecrease, { PriceDecreaseProps } from '../';

const LiveViewersWithRequiredProps: FC<Required<PriceDecreaseProps>> = PriceDecrease;

describe('<Radial />', () => {
  test('Radial props with mandatory props', async () => {
    const tree = renderer.create(<PriceDecrease />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('Radial props with all props as required', async () => {
    const tree = renderer.create(<LiveViewersWithRequiredProps duration={1000} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
