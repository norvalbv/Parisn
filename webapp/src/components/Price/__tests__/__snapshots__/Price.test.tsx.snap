// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Radial /> Radial props with all props as required 1`] = `
<div>
  PriceDecrease
</div>
`;

exports[`<Radial /> Radial props with mandatory props 1`] = `
<div>
  PriceDecrease
</div>
`;
